plugins/gopass.yaml

```
@@ -0,0 +1,22 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: gopass
+spec:
+  version: "v0.0.3"
+  platforms:
+    - selector:
+        matchExpressions:
+          - {key: os, operator: In, values: [darwin, linux]}
+      uri: https://github.com/gopasspw/kubectl-gopass/archive/v0.0.3.zip
+      sha256: "ae4d2aa8ec061aa6916b2120fa3caf48d1cee328bab7cd263a03f8b1aeec22f8"
+      files:
+        - from: "/kubectl-gopass-*/kubectl-gopass"
+          to: "."
+      bin: "./kubectl-gopass"
+  shortDescription: Imports secrets from gopass
```

```suggestion
  shortDescription: Imports Secrets from gopass
```
Making it a capital `S` signals that this is about kubernetes Secret resources. Or maybe  even

> Imports Secret resources from gopass